crates/router/src/configs/defaults.rs

```diff
@@ -8239,6 +8239,139 @@ impl Default for super::settings::RequiredFields {
                                         common : HashMap::new(),
                                     }
                                 ),
+                                (
+                                    enums::Connector::Klarna,
+                                    RequiredFieldFinal {
+                                        mandate: HashMap::new(),
+                                        non_mandate: HashMap::from([
+                                            (
+                                                "shipping.address.first_name".to_string(),
+                                                RequiredFieldInfo {
+                                                    required_field: "shipping.address.first_name".to_string(),
+                                                    display_name: "given_name".to_string(),
```

Comment:
We need to use our terminology here, so whatever is used as display_name for this field across SDK Should be consistent

Reply:
Please do this at all the required places

Reply:
Also confirm from SDK do they need this field in Express checkout flow at someplace.